src/js/App.js

```diff
@@ -1,70 +1,68 @@
-import React, { Component } from "react"
-import "../styles/App.css"
-import { STEPS } from "../config"
-import { Arrow } from "./components"
-import { Step } from "./pages"
+import React, { Component } from "react";
+import "../styles/App.css";
+import { STEPS } from "../config";
+import { Arrow } from "./components";
+import { Step } from "./pages";
 
 class App extends Component {
   constructor(props, context) {
-    super(props, context)
+    super(props, context);
     this.state = {
       activeStep: STEPS[0]
-    }
+    };
   }
 
   render() {
     return (
       <div className="App">
         <Arrow
+          show={this.getStepIdx() > 0}
```

Comment:
This is an interesting way of handling the showing and hiding of the arrow functions. Typically you would want to implement this with an inversion of control so that the `Arrow` component didn't need to handle its own visibility. This sounds kind of backwards but if the logic is in the parent component usually the rendering logic should be there as well.

React gives is a really easy way to conditionally render small pieces of ui. A common approach is below.

```suggestion
{this.getStepIdx() && <Arrow  direction="left" onClick={this.changeStep.bind(this)}/>}
```

Reply:
The `&&` is just a shorthand ternary style "display if truthy" statement. You could also use `this.getStepIdx() ? <Arrow /> : null`